assets/js/stripe-payment-request.js

```diff
@@ -87,20 +87,21 @@ jQuery( function( $ ) {
 			var email    = source.owner.email;
 			var phone    = source.owner.phone;
 			var billing  = source.owner.address;
+			var name     = source.owner.name;
 			var shipping = evt.shippingAddress;
 			var data     = {
 				_wpnonce:                  wc_stripe_payment_request_params.nonce.checkout,
-				billing_first_name:        source.owner.name.split( ' ' ).slice( 0, 1 ).join( ' ' ),
-				billing_last_name:         source.owner.name.split( ' ' ).slice( 1 ).join( ' ' ),
+				billing_first_name:        null !== name    ? name.split( ' ' ).slice( 0, 1 ).join( ' ' ) : '',
+				billing_last_name:         null !== name    ? name.split( ' ' ).slice( 1 ).join( ' ' ) : '',
```

Comment:
Can you push the question mark back to one space as they don't need to be aligned with the rest down the line.